docs: compatibility guide (#1567)
aws/aws-advanced-jdbc-wrapper: docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

In addition to the built-in plugins, you can also create custom plugins more suitable for your needs.
For more information, see [Custom Plugins](../development-guide/LoadablePlugins.md#using-custom-plugins).

> Some plugins, database types and database URL types may be incompatible. More information available within this [compatibility guide](./Compatibility.md).

Reviewer: ```suggestion
Some plugins, database types and database URL types may be incompatible. For more information see the [compatibility guide](./Compatibility.md).
```
